src/service/Data.tsx

```diff
@@ -31,12 +31,10 @@ const getLastName = (modite: Modite): string => {
 const sortRecords = (records: (Modite | Project)[]): void => {
   if (records.length) {
     records.sort((prev: Modite | Project, next: Modite | Project) => {
-      const prevName = getLastName(prev)
-      const nextName = getLastName(next)
+      const prevName = getLastName(prev).toLowerCase();
+      const nextName = getLastName(next).toLowerCase();
 
-      if (prevName < nextName) return -1
-      if (prevName > nextName) return 1
-      return 0
+      return prevName > nextName ? 1 : nextName > prevName ? -1 : 0;
```

Comment:
it's generally easier to read the code above than a double ternary. I'd prefer code that focuses on humans 😄

Reply:
hahaha, ok... will change it back :P

Reply:
Done @grgur